lib/srv/regular/sshserver.go

```diff
@@ -457,6 +458,13 @@ func SetOnHeartbeat(fn func(error)) ServerOption {
 	}
 }
 
+func SetKubernetesClusters(clusters []string) ServerOption {
```

Comment:
How is this used on a SSH server?

Reply:
Added a comment.
`lib/srv/regular.Server` manages proxy heartbeats. Heartbeats is how a proxy announces its supported k8s clusters to the rest of the teleport cluster.
Logically, heartbeats are a separate feature from SSH serving, but untangling these would be a noisy refactoring.

Reply:
Makes sense.